src/components/desktop/Header.js

```diff
@@ -27,12 +30,37 @@ const StyledHeader = styled.header`
       li {
         display: inline-grid;
         place-items: center;
```

Comment:
```suggestion
        display: inline-flex;
        justify-content: center;
        align-items: center;
```

This should solve the weird bug we have with the extra pixel at the top :)